### ISSUE TITLE
Homepage - Search field is missing an accessible label

### ISSUE THREAD

**Issue by ewlarson** [geobtaa/geoportal#514]

**Comment by ewlarson** [comment]
https://github.com/github/auto-complete-element

**Comment by ewlarson** [comment]
Issue opened in GBL for migrating from Twitter Typeahead to GitHub AutoCompleteElement:
https://github.com/geoblacklight/geoblacklight/issues/1292
